### PR TITLE
Add #undistro to channels.yaml

### PR DESCRIPTION
**Pull request by felipeweb**
Adding a Slack channel in accordance with the guidelines. 

[UnDistro](https://undistro.io) is a Kubernetes distribution.  The channel will be used to discuss UnDistro with end users and contributors, answer questions, gather bugs and feature requests, etc.

### CODE REVIEW

File: communication/slack-config/channels.yaml [hunk 1]
```diff
@@ -389,9 +389,11 @@ channels:
   - name: ug-big-data
     id: C0ELB338T
   - name: ug-vmware
+  - name: undistro
   - name: velero-dev
   - name: velero-users
     id: C6VCGP4MT
+  - name: velero
```
Comment (coderanger) [inline communication/slack-config/channels.yaml:396]: This isn't supposed to be here :)

**Review by coderanger — CHANGES_REQUESTED** [review]

File: communication/slack-config/channels.yaml (hunk 1 above)
Comment (felipeweb) [reply communication/slack-config/channels.yaml]: order fixed

Comment (coderanger) [reply communication/slack-config/channels.yaml]: No, I mean that channel has been removed, your rebase got confused.

Comment (felipeweb) [reply communication/slack-config/channels.yaml]: now it's removed

**Comment by felipeweb** [comment]
@DylanGraham   I guess the correct command is approve

**Pull request merged**